### PR TITLE
Add CI support for front end

### PR DESCRIPTION
**Pull request by ncloudioj**

### CODE REVIEW

**Comment by ncloudioj** [comment]
r? @rlr & @tkiethanom

We're reusing a single TravisCI for now. It might slow down the process as it'll run tests for both backend and frontend. The speed looks good to me though. We can split it up later if it gets unacceptable.

@tkiethanom I've renamed the "routes.js" to "Routes.js" otherwise it complained the "file is missing". Makes sense to you?

File: front_end/package.json [hunk 1]
```diff
@@ -81,5 +81,8 @@
     "url-loader": "~0.5.6",
     "webpack": "^1.12.0",
     "webpack-dev-server": "~1.10.1"
+  },
+  "scripts": {
+    "test": "./node_modules/.bin/karma start --single-run --browsers Firefox"
```
Comment (rlr) [inline front_end/package.json]: nice. :+1:

**Comment by rlr** [comment]
`Firefox 31.0.0 (Linux 0.0.0): Executed 9 of 9 SUCCESS (0.134 secs / 0.083 secs)`

I guess travis needs to update their firefox.

**Comment by rlr** [comment]
Hey it worked. r+ from me.

**Comment by ncloudioj** [comment]
Thank you sir! @rlr 

We could do an extra "apt-get install firefox && go get a coffee", but I am an inpatient guy:).

**Comment by oyiptong** [comment]
In case Firefox lags behind too much, you can check out our automation at https://github.com/mozilla/remote-newtab/blob/f7acc299b5911ed828c5e8b22a7e7b69f129b655/.travis.yml

`./node_modules/.bin/mozilla-download ` is installed through an npm dependency: https://github.com/mozilla/remote-newtab/blob/f7acc299b5911ed828c5e8b22a7e7b69f129b655/package.json#L36

We download the latest versions of Firefox based on our choice of channels.

**Comment by ncloudioj** [comment]
Nice, that's very helpful. Thanks @oyiptong.

**Comment by tkiethanom** [comment]
Do you remember what the error was that you needed to capitalized Routes.js? I think it might be a problem with the main.js referencing Routes.js instead of routes.js.

**Comment by ncloudioj** [comment]
@tkiethanom "file or diretory './Routes.js' is missing". yes,  it was from the `main.js`. You can also fix it in `main.js`, totally up to you.

**Comment by tkiethanom** [comment]
Yea just update main.js as part of the pull request. And revert it back to lowercase routes.js

**Comment by ncloudioj** [comment]
@tkiethanom no problem. will do.

**Pull request closed without merging**